### PR TITLE
Feature/create room

### LINKED ISSUE #10: Page for creating a room
Including a (Google) map, date range, description, number of beds

### PR DESCRIPTION
**Pull request by lyuboxin**
resolved #10 Create addroom page

### CODE REVIEW

File: addroom.html [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+      form_login.onsubmit = function(e) {
+        e.preventDefault();
+        var housename = document.getElementById("name").value;
+        var streetname = document.getElementById("street").value;
+        var cityname = document.getElementById("City").value;
+        var provincename = document.getElementById("Provinces").value;
+        var postalcode = document.getElementById("code").value;
+        $.ajax({
+          type: "POST",
+          url: "http://192.168.1.104:8000/house/create",
```
Comment (xingweitian) [inline addroom.html]: This links seems a test link, it should be the link of cs server, right?

File: addroom.html [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+            place_id: placeid,
+            house_address: streetname,
+            house_city: cityname,
+            house_province: provincename,
+            house_postcode: postalcode,
+            longitude: lng,
+            latitude: lat
+          },
+          success: function(msg) {
+            console.log("hello");
```
Comment (xingweitian) [inline addroom.html]: Since you are trying to merge to master, I am not sure logging "hello" is suitable here.

**Comment by lyuboxin** [comment]
I have tried to separate JS code from html page, but I came cross an error about google map api. Therefore, I put them together in html page.

**Review by xingweitian — CHANGES_REQUESTED** [review]

File: addroom.html (hunk 1 above)
Comment (lyuboxin) [reply addroom.html]: Yes, you're right. I forgot to modify the url. I will change it to the correct link

File: addroom.html [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+          },
+          fail: function() {
+            console.log("add failed");
```
Comment (xingweitian) [inline addroom.html:171]: Same here, try to make log info more specific.

Comment (lyuboxin) [reply addroom.html:171]: I have fixed it. Please check

**Review by xingweitian — APPROVED** [review]

**Review by zhangjiangqige — APPROVED** [review]
2 things left for the future

1. find a way to move the js to a .js file
2. add three new parameters: number of beds, and range of date

Maybe we can also add a `defer` in the <script> of this new .js file. I doubt the error you mentioned has something to do with this keyword.

**Comment by zhangjiangqige** [comment]
BTW it's a bad practice to use the same commit message all along, as described in our wiki.

**Comment by lyuboxin** [comment]
> 2 things left for the future
> 
> 1. find a way to move the js to a .js file
> 2. add three new parameters: number of beds, and range of date
> 
> Maybe we can also add a `defer` in the <script> of this new .js file. I doubt the error you mentioned has something to do with this keyword.

I will address these issues and try to solve them.

**Comment by lyuboxin** [comment]
> BTW it's a bad practice to use the same commit message all along, as described in our wiki.

I see. I will pay attention to it

**Pull request merged**